### PR TITLE
POC: Added extra package.json with dependencies

### PR DESCRIPTION
**Pull request by iberdinsky-skilld**
In this PR component has own package.json file

1. with own dependencies
2. with version
3. with build script

For building used common `vite.components-config.js` 

All css/js assets now builded with new script `yarn bc`. They are not gitignored anymore. And keep everything inside.

`vite.config.js` works as before only for building theme own assets/fonts/images

This PR will allow to share components between projects much easier. 

**Update**

Added separation in dependant library. So now `yarn bc` command moves imported libraries into component `/libraries` dir. 

**Update 2**

Added lerna monorepo manager. And extra test PR https://github.com/skilld-labs/ui-patterns-kaizen/pull/7/files 
_removed_ lerna_

**Update 3**
- Added VScode variables autocomplete https://open-props.style/#autocomplete
- Added https://open-props.style/ with https://open-props.style/#jitprops jitProps support
- updated .breakpoints.yml with default values from open-props, with ability to override. 
- updated postcss config to use ES module. 
- Created a-text + a-button components for design for testing purposes https://www.figma.com/file/MmteI2ZZUPLW63yidyzl9A/Portfolio-Template-(Community)?node-id=101%3A412

**Update 3**

Added menu component, with preview for both Drupal and Storybook

### CODE REVIEW

File: web/themes/custom/caesar/templates/patterns/atoms/button/package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@caesar/a-button",
+  "version": "1.0.0",
+  "license": "MIT",
+  "dependencies": {
+    "@splidejs/splide": "^4.1.4"
```
Comment (andypost) [inline web/themes/custom/caesar/templates/patterns/atoms/button/package.json:6]: Please remove it, there's no wait to track dependencies for components which use it

Also no way to maintain (bump) components which using it

And there's no way to document how inter-dependencies should be upgraded according to semver.org

**Review by andypost — CHANGES_REQUESTED** [review]
Do not add versioning for each component as it leads to unpredictable maintenance costs

File: web/themes/custom/caesar/templates/patterns/atoms/button/package.json (hunk 1 above)
Comment (iberdinsky-skilld) [reply web/themes/custom/caesar/templates/patterns/atoms/button/package.json:6]: this is new branch based on current one with bumping of version
https://github.com/skilld-labs/ui-patterns-kaizen/pull/7/files

more info:
https://github.com/lerna/lerna/tree/main/commands/version#semver-bump

Comment (iberdinsky-skilld) [reply web/themes/custom/caesar/templates/patterns/atoms/button/package.json:6]: Dependencies not changed yet. Gonna research it later.

**Review by gaydabura — CHANGES_REQUESTED** [review]
Atoms **must not** be a patterns

**Pull request merged**